Makefile

```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GO           ?= GO15VENDOREXPERIMENT=1 go
+GO           := go
```

Comment:
Please keep the option to overwrite the GO endpoint and use `?=`.

Reply:
✓